helm/external-dns-app/templates/ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.crd.install }}
```

Comment:
Is there any reason to move this resource outside the `crd` directory? as far as I know, there is no change in behavior. 
If that's the case, I'd suggest to move it back to the crd jobs folder to maintain them separated (eventually we'd like to move the crd job to an chart dependency).

Reply:
good point, done